gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -368,8 +419,22 @@ nnstreamer_filter_validate (const GstTensorFilterFramework * tfsp)
 int
 nnstreamer_filter_probe (GstTensorFilterFramework * tfsp)
 {
+  GstTensorFilterFrameworkInfo info;
+  GstTensorFilterProperties prop;
+  char *name;
+
   g_return_val_if_fail (nnstreamer_filter_validate (tfsp), FALSE);
-  return register_subplugin (NNS_SUBPLUGIN_FILTER, tfsp->name, tfsp);
+
+  if (checkGstTensorFitlerFrameworkVersion (tfsp->version,
+          GST_TENSOR_FILTER_FRAMEWORK_V0)) {
+    name = tfsp->name;
+  } else {
```

Comment:
How about checking version 1 as well, using `else if` ?

Reply:
This is done in `nnstreamer_filter_validate()`. If the code reaches this location, it is assumed that the passed framework is a valid object.